src/components/Loading/Loading.js

```diff
@@ -52,7 +53,21 @@ export default class Loading extends React.Component {
         aria-live={active ? 'assertive' : 'off'}
         className={loadingClasses}>
         <svg className={`${prefix}--loading__svg`} viewBox="-75 -75 150 150">
-          <circle cx="0" cy="0" r="37.5" />
+          <title>Loading</title>
+          {componentsX && small ? (
```

Comment:
As a style convention, do we prefer the ternary pattern or `&&` chain pattern, like:

```js
{componentsX && small && (
  <circle />
)}
```

Reply:
While I prefer chaining personally, I think this project has accept either approach.